core/src/main/java/com/automation/remarks/video/recorder/ffmpeg/LinuxFFmpegRecorder.java

```diff
@@ -6,7 +6,7 @@
 public class LinuxFFmpegRecorder extends FFMpegRecorder {
     @Override
     public void start() {
-        String display = ":0.0";
+        String display = System.getProperty("recorder.display", ":0.0");
```

Comment:
I think that to be consistent we should use configuration class

https://github.com/SergeyPirogov/video-recorder-java/blob/master/core/src/main/java/com/automation/remarks/video/recorder/VideoConfiguration.java